### PR TITLE
cleanup: enhance docs ahead of full publication

### PR DESCRIPTION
**Pull request by douglas-reid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,16 +35,30 @@ Initial support is offered for the following (with more to follow soon):
 - LLMs
   - An adapter is provided for Steamship's OpenAI integration (`steamship_langchain.llms.OpenAI`)
   - An adapter is provided for *caching* LLM calls, via Steamship's Key-Value store (`SteamshipCache`) 
+- Document Loaders
+  - An adapter for loading Steamship Files into Documents is provided (`steamship_langchain.document_loaders.SteamshipLoader`)
```
Comment (dkolas) [inline README.md:39]: Is it Files -> Documents or Documents -> Files?  Or both?

File: README.md [hunk 2]
```diff
@@ -32,16 +35,30 @@ Initial support is offered for the following (with more to follow soon):
...
   - Search:
     - An adapter is provided for Steamship's SERPAPI integration (`SteamshipSERP`)
 - Memory
   - Two adapters that provide persistent conversation memory:
     - Complete Memory (`steamship_langchain.memory.ConversationBufferMemory`)
     - Windowed Memory (`steamship_langchain.memory.ConversationBufferWindowMemory`)
+- VectorStores
+  - An adapter is provided for a persistent VectorStore (`steamship_langchain.vectorstores.SteamshipVectorStore`)
+- Text Splitters
+  - A splitter for Python code, based on the AST, is provided (`steamship_langchain.python_splitter.PythonCodeSplitter`)
```
Comment (dkolas) [inline README.md:50]: A note here about how it's different than the one you said was in langchain?

File: docs/adapters/llms/getting_started.ipynb [hunk 3]
```diff
@@ -77,9 +77,7 @@
      "text": [
       "Completion: \n",
       "\n",
-      "Why did the chicken cross the road?\n",
-      "\n",
-      "To get to the other side!\n"
+      "A man walks into a bar and asks for a beer. The bartender says \"You're out of luck. We've been closed for fifteen minutes.\"\n"
```
Comment (dkolas) [inline docs/adapters/llms/getting_started.ipynb:80]: Not a big deal, but do you want to install nbstripout to clear the output from the notebooks before committing?  Maybe not if you think there's value in seeing the output by someone browsing the repo.

**Review by dkolas — APPROVED** [review]
Looks good! Only minor comments

File: docs/adapters/llms/getting_started.ipynb (hunk 3 above)
Comment (douglas-reid) [reply docs/adapters/llms/getting_started.ipynb:80]: I think we need the output for both repo browsing and the docs site generation. Without the output, the rendered HTML will not show the expected results.

File: README.md (hunk 1 above)
Comment (douglas-reid) [reply README.md:39]: Files -> Documents.

Comment (douglas-reid) [reply README.md:39]: maybe i should say `exporting` ?

File: README.md (hunk 2 above)
Comment (douglas-reid) [reply README.md:50]: Good idea.

**Comment by douglas-reid** [comment]
Thanks for the review @dkolas! I'm gonna go ahead and merge. Once the site is fully up, we can take a fuller editing pass.

**Pull request merged**